task_processing/plugins/mesos/execution_framework.py

```diff
@@ -530,10 +530,7 @@ def statusUpdate(self, driver, update) -> None:
         task_id = update.task_id.value
         task_state = str(update.state)
 
-        log.info("Task update {update} received for task {task}".format(
-            update=task_state,
-            task=task_id
-        ))
+        log.info(f'Task update {task_state} received for task {task_id} at time {update.timestamp}')
```

Comment:
hmm... update.timestamp is when the update was generated. Maybe reword this or change it to time.time